config.go

```diff
@@ -168,6 +168,11 @@ type Config struct {
 	// the server. If this is unacceptable to the server then it may abort
 	// the handshake.
 	GetClientCertificate func(*CertificateRequestInfo) (*tls.Certificate, error)
+
+	// SkipHelloVerify, if true and when acting as server, allow client to
+	// skip hello verify phase and receive ServerHello after initial ClientHello.
+	// This have implication on DoS attack resistance.
+	SkipHelloVerify bool
```

Comment:
I would propose the rename the option to InsecureSkipVerifyHello or something along those line, to keep it aligned with tls.Config's InsecureSkipVerify options.